reference/shared/office.context.auth.getAccessTokenAsync.md

```diff
@@ -74,7 +76,7 @@ The following actions are available during authentication.
 ### callback method
 When the function you passed to the  _callback_ parameter executes, it receives an [AsyncResult](../../reference/shared/asyncresult.md) object that you can access from the callback function's only parameter.
 
-In the callback function passed to the  **getAccessTokenAsync** method, you can use the properties of the **AsyncResult** object to return [add-in identity objects](#Identity-Objects).
+In the callback function passed to the  **getAccessTokenAsync** method, you can use the properties of the **AsyncResult** object to access the [add-in identity objects](#Identity-Objects).
```

Comment:
And I would also move that section way up (to here), as opposed to being at the bottom of the document